src/wallet/Transfer.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import numeral from 'numeral';
+import Icon from '../widgets/Icon';
+import Header from '../app/Header';
+
+@connect(
+  state => ({
+    auth: state.auth,
+  })
+)
+
```

Comment:
@adcpm `connect` is a Higher Order Component which means is containing the whole class like a function. As we don't leave a space in this example:
```
function something(

{ foo: 'bar' }
) { ...
```
IMHO we also shouldn't leave there because empty lines are used for separating conceptually independent parts but `connect` and the class it's containing are tightly coupled.